scripts/babel/transform-event-target.js

```diff
@@ -0,0 +1,38 @@
+const warningDevCheckTemplate = `
+  if (process.env.NODE_ENV !== 'production') {
+    NODE;
+  }
+`.trim();
+
+const visited = 'visitedByEventTargetTransformer';
+const calleeProperty = 'dispatchEvent';
+const calleeObject = 'eventTarget';
+const client = 'client';
```

Comment:
Minor nit: I know we want to enforce that it's called on the client, but the name `client` isn't necessarily the name of a local variables 😅  so I think we'll have to ignore the name of the `callee`